packages/common/src/utils/internal/userAppResources.ts

```diff
@@ -0,0 +1,174 @@
+// These Types and Functions can land in rest-js
+// a friendly name for the scope of the resource
+// the API deals with clientIds, which are
+// listed here https://confluencewikidev.esri.com/pages/viewpage.action?pageId=50561461
+// Generally speaking, the UserResourceScope is a downcased version of the
+// clientId. Hub is unique b/c Sites have their own clientId,
+
+import { IRequestOptions, request } from "@esri/arcgis-rest-request";
+import { failSafe } from "../fail-safe";
+import { getObjectSize } from "../getObjectSize";
+
+// Defines the access for the resource
+export type UserResourceAccess =
+  | "userappprivate" // default
+  | "userprivateallapps" // readable by any platform app, private to user
+  | "allorgusersprivateapp" // only if user is public or org
```

Comment:
Please include a link to the docs unless there's a very specific reason not to. I was confused by this and I know for a fact that future devs will also be confused.

Reply:
There is a link to the cf page just above this in the comment for the module. I also marked everything in this file as `@private` so it's kept out of the docs - as per chat with AGO portal team